framerail/.gitignore

```diff
@@ -6,3 +6,5 @@ node_modules
 .env
 .env.*
 !.env.example
+
+/src/types/index.ts
```

Comment:
Why are we ignoring the generated file? Especially since you're adding it in this PR